model/ConsumerService.php

```diff
@@ -64,14 +66,35 @@ public function getConfiguration($iss = '')
             $res = $instance->getPropertiesValues([
                 self::PROPERTY_KEY,
                 self::PROPERTY_SECRET,
+                self::PROPERTY_ENTRY_POINT,
             ]);
 
             $config = [
                 self::PROPERTY_KEY => $res[self::PROPERTY_KEY][0]->literal,
                 self::PROPERTY_SECRET => $res[self::PROPERTY_SECRET][0]->literal,
+                self::PROPERTY_ENTRY_POINT => $res[self::PROPERTY_ENTRY_POINT][0]->literal,
             ];
         }
 
         return $config;
     }
+
+    /**
+     * @return array
+     */
+    public function getAvailableEntryPoints()
```

Comment:
@chrisphpgarcia As far as I know standard endpoint system is focused more on end-user flow managing, based on ACL. Instead of this case, where I wanted to handle openId specific situation. May be just miss something with regular endpoints?